src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/eventDetailsBuilder.hpp

```diff
@@ -229,6 +231,23 @@ class TEventDetailsBuilder final : public AbstractHandler<std::shared_ptr<TScanC
                 auto vulnerabilityDetection = PolicyManager::instance().getVulnerabilityDetection();
                 ecsData["wazuh"]["cluster"]["name"] = data->clusterName();
                 ecsData["wazuh"]["schema"]["version"] = WAZUH_SCHEMA_VERSION;
+                switch (data->affectedComponentType())
+                {
+                    case AffectedComponentType::Package:
+                        ecsData["wazuh"]["vulnerability"]["source"] = TGlobalData::instance().adpDescriptions().at(
```

Comment:
This should be vulnerability.scanner.source

Reply:
I didn't upload the changes we discussed earlier yet